<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>921</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>921</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №6</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Алгоритм заполнения (закраски)&lt;br/&gt;с затравкой&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>175</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>175</height>
         </size>
        </property>
        <property name="title">
         <string>Цвет</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет рисования</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет фона</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changePenColorBtn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changeBackgroundColorBtn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QLabel" name="PenColor">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="backgroundColor">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: white</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>360</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Точки</string>
         </attribute>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>301</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавление затравки&lt;br/&gt;(либо правой кнопкой мыши)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>40</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTextEdit" name="seedX">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="seedY">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="addSeedBtn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить затравку</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>281</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавление точки&lt;br/&gt;(либо левой кнопкой мыши)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>200</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTextEdit" name="pointX">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="pointY">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>230</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="addPointBtn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить точку</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Эллипс</string>
         </attribute>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Координаты центра</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="ellipseCenterX">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="ellipseCenterY">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>40</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Полуоси</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Малая</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>160</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Большая</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTextEdit" name="smallAxis">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="bigAxis">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>190</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="addEllipseBtn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>270</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить эллипс</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="paintShapeBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Закрасить фигуру</string>
         </property>
        </widget>
        <widget class="QPushButton" name="closeShapeBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Замкнуть фигуру</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Очистить плоскость</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Время выполнения</string>
         </property>
        </widget>
        <widget class="QLabel" name="timeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0 мс</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>С задержкой</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Загрузить данные</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
